### ISSUE TITLE
Train acc and val acc are not improving

### ISSUE THREAD

**Issue by navidstuv** [tkipf/gcn#38]
Hi,

I have a dataset, consisting of 2500 nodes and feature vectors (X (2500,10)) with the values between zero and one (sum of each row is one,  e.g., sum(X(i,:))=1). The training and validation accuracy are stuck on a value without any change. What do you think the problem might be?

Epoch: 0037 train_loss= 1.32362 train_acc= 0.62857 val_loss= 1.45957 val_acc= 0.51000 time= 0.31915
Epoch: 0038 train_loss= 1.28895 train_acc= 0.62857 val_loss= 1.45154 val_acc= 0.51000 time= 0.31516
Epoch: 0039 train_loss= 1.28910 train_acc= 0.62857 val_loss= 1.44433 val_acc= 0.51000 time= 0.31915
Epoch: 0040 train_loss= 1.26195 train_acc= 0.62857 val_loss= 1.43808 val_acc= 0.51000 time= 0.31615
Epoch: 0041 train_loss= 1.25370 train_acc= 0.62857 val_loss= 1.43273 val_acc= 0.51000 time= 0.31815
Epoch: 0042 train_loss= 1.24904 train_acc= 0.62857 val_loss= 1.42827 val_acc= 0.51000 time= 0.32114
Epoch: 0043 train_loss= 1.23366 train_acc= 0.62857 val_loss= 1.42465 val_acc= 0.51000 time= 0.31815
Epoch: 0044 train_loss= 1.22609 train_acc= 0.62857 val_loss= 1.42184 val_acc= 0.51000 time= 0.31815
Epoch: 0045 train_loss= 1.22172 train_acc= 0.62857 val_loss= 1.41968 val_acc= 0.51000 time= 0.31416
Epoch: 0046 train_loss= 1.21135 train_acc= 0.62857 val_loss= 1.41816 val_acc= 0.51000 time= 0.31815
Epoch: 0047 train_loss= 1.20308 train_acc= 0.62857 val_loss= 1.41732 val_acc= 0.51000 time= 0.31715
Epoch: 0048 train_loss= 1.19649 train_acc= 0.62857 val_loss= 1.41703 val_acc= 0.51000 time= 0.31815
Epoch: 0049 train_loss= 1.19196 train_acc= 0.62857 val_loss= 1.41708 val_acc= 0.51000 time= 0.31715
Epoch: 0050 train_loss= 1.18271 train_acc= 0.62857 val_loss= 1.41766 val_acc= 0.51000 time= 0.31516
Epoch: 0051 train_loss= 1.18121 train_acc= 0.62857 val_loss= 1.41835 val_acc= 0.51000 time= 0.31615
Epoch: 0052 train_loss= 1.17401 train_acc= 0.62857 val_loss= 1.41921 val_acc= 0.51000 time= 0.31715
Epoch: 0053 train_loss= 1.17111 train_acc= 0.62857 val_loss= 1.42014 val_acc= 0.51000 time= 0.31715

**Comment by tkipf** [comment]
Hard to say. Have you tried increasing model capacity (more hidden units)
and disable regularizers to see if you can at least overfit on the training
set? If you can’t, then a) GCNs are not sensitive to what you try to fit or
b) something is not right with your data/task.

**Comment by navidstuv** [comment]
I increased model capacity and disabled the regularizers, the accuracy values are exactly the same as before.:(
I expect at least values change!!!!

**Comment by jjjj0x** [comment]
I met the same problem recently. Have you solve this problem already? If you do, please help me.
